Solutions/FalconFriday/Analytic Rules/Disable or Modify Tools.yaml

```diff
@@ -0,0 +1,47 @@
+id: 20d52a04-b5d8-402d-88e2-7929d12cbdcd
+name: Disable or Modify Tools
+description: |
+  This detection watches the commandline logs for known commands that are used to disable the Defender AV. This is based on research performed by @olafhartong on a large sample of malware for varying purposes. 
+  Note that this detection is imperfect and is only meant to serve as basis for building a more resilient detection rule. 
+  Some software behaves weirdly, we've at least identified that JetBrains and Android Studio are behaving oddly. Not sure why, yet. Perhaps I should stop using JetBrains?
+  Make the detection more resilient, currently the order of parameters matters. You don't want that for a production rule. See blogpost for more resilience considerations.
+  The current approach can easily be bypassed by not using the powershell.exe executable. Consider adding more ways to detect this behavior.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceProcessEvents
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562.001
+query: |
+  let defendertampering=dynamic(["Set-MpPreference -DisableRealtimeMonitoring $true","sc stop WinDefend","sc delete WinDefend","Set-MpPreference -DisableBehaviorMonitoring $true","Set-MpPreference -ExclusionProcess", "Set-MpPreference -ExclusionExtension dll","net stop security center"]);
+  DeviceProcessEvents
+  | where ProcessCommandLine has_any (defendertampering)
+  // If you have a lot of false positives coming from JetBrains, you can use the line below. 
+  //| where InitiatingProcessFolderPath !startswith @"c:\program files\jetbrains\" and InitiatingProcessVersionInfoProductName !~ ("Android Studio")
```

Comment:
Format output by adding `project` or `summarize`

Reply:
See previous comments as well. Leaving this issue unresolved to have a disscussion. 

IMHO, projecting/summarizing the info is a task for the analyst implementing this query. I don't want to throw away info which might be useful for the analyst. Since every organization has a different way of triaging incidents, they might need different columns/info. 

Can you please let me know if you agree/disagree?

Reply:
I am fine with this suggestion, its not mandatory. You can always revisit the output formatting based on community feedback after its available. 
Most of the analytic queries in GitHub, we try to limit the important fields to display as sometimes displaying long schema is not helpful for analysts in UI on the first view.